src/main/java/com/thebois/views/ColonyView.java

```diff
@@ -1,42 +1,110 @@
 package com.thebois.views;
 
+import java.util.HashMap;
+import java.util.Map;
+
 import com.badlogic.gdx.graphics.Color;
-import com.badlogic.gdx.graphics.glutils.ShapeRenderer;
+import com.badlogic.gdx.graphics.Pixmap;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Batch;
 
 import com.thebois.models.beings.IBeing;
 import com.thebois.models.beings.IBeingGroup;
+import com.thebois.models.beings.roles.RoleType;
 
 /**
  * Used to display all pawns.
  */
 public class ColonyView implements IView {
 
+    private static final Map<RoleType, Color> ROLE_COLORS;
+    /**
+     * How many times bigger the textures should be relative to their displayed size.
+     *
+     * <p> E.g. a factor of 2 means the textures are created at 2x the resolution they are being
+     * displayed at. </p>
+     */
+    private static final int TEXTURE_SUPER_SAMPLING_FACTOR = 8;
+
+    static {
+        ROLE_COLORS = new HashMap<>();
+        ROLE_COLORS.put(RoleType.BUILDER, Color.LIGHT_GRAY);
+        ROLE_COLORS.put(RoleType.FARMER, Color.GREEN);
+        ROLE_COLORS.put(RoleType.FISHER, Color.BLUE);
+        ROLE_COLORS.put(RoleType.GUARD, Color.BLACK);
+        ROLE_COLORS.put(RoleType.LUMBERJACK, Color.BROWN);
+        ROLE_COLORS.put(RoleType.MINER, Color.GRAY);
+    }
+
+    private final Color beingColor = Color.WHITE;
```

Comment:
`defaultPawnColor` or if we have a default role `laborerColor`

Reply:
Good idea, once I've added a role for idle pawns I'll change this too.